pages/summary-confirmation.js

```diff
@@ -169,7 +169,6 @@ const ApplicationComplete = (props) => (
       <Heading as="h2" size="LARGE">
         {props.t("Find out here what you can do to prepare")}
       </Heading>
-      <HintText mb={3}>{props.t("All links open in a new window")}</HintText>
```

Comment:
The GDS guidelines suggests (https://design-system.service.gov.uk/styles/typography/#links):

> If you’re displaying lots of links together and want to save space and avoid repetition, consider doing both of the following:
> 
> - adding a line of text before the links saying ‘The following links open in a new tab’
> 
> - including `<span class="govuk-visually-hidden">(opens in new tab)</span>` as part of the link text, so that part of the link text is visually hidden but still accessible to screen readers
>